### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Mikey1-nuel**
## Math magicians: full website.

**In this project we:**

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Added our own styles to improve the look and feel.

- We created a route for each page.
- Made sure that navigation links for all 3 routes are displayed on each page.
- Used React Router to make the routes work.

### CODE REVIEW

File: src/App.css
Comment (KDlamini) [inline src/App.css:1]: image

Kindly make sure that you do not have any broken links on the README. The URL to the issues page is correct but the way you added the link compromised the URL. 

- When you add a link, the text should be in the square brackets `[issues]` and the URL in the parenthesis `(URL)`.

Comment (KDlamini) [inline src/App.css:1]: As in the wireframe kindly make sure that the calculator page has a heading.

image

image


Kindly make sure that you do not disable any linter rules. Linters are meant to help you follow the best practices. Please find a way to implement your code without disabling linter rules.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Mikey1-nuel 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a working Math Magicians website using React. 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️ Good PR title and description
✔️ The website consists of 3 pages: `Home`, `Calculator`, and `Quote`
✔️ Navigation bar menu directs to each dedicated page route

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KDlamini** [comment]
Kindly make sure that you do not disable any linter rules. Linters are meant to help you follow the best practices. Please find a way to implement your code without disabling linter rules.

image

**Review by addod19 — APPROVED** [review]
### Hi @Mikey1-nuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job addressing all the comments from the previous review :rocket: awesome


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**